src/dcd/client/client.d

```diff
@@ -35,6 +35,20 @@ import dcd.common.dcd_version;
 import dcd.common.socket;
 
 int main(string[] args)
+{
+	try
+	{
+		return runClient(args);
+	}
+	catch (Exception)
+	{
+		return -1;
```

Comment:
Please log the exception to console, otherwise errors in dcd-client will get undebugable

Reply:
Would ```writeln(e.msg)``` be sufficient or do you need the stacktrace (e.info).
Also what do you prefer, stdout or stderr?

Reply:
it should contain the stacktrace and definitely stderr

Reply:
fixed

Reply:
Fixed